spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -309,10 +278,47 @@
     end
   end
 
-  def stub_controller_with_exception
+  let(:simulated_error_class) do
+    Class.new(StandardError)
+  end
+
+  def define_controller_for_resource_with_strong_parameters(
```

Comment:
Assignment Branch Condition size for define_controller_for_resource_with_strong_parameters is too high. [18.06/15]